### PR TITLE
Update images and labels in Dockerfile.rhel

### PR DESCRIPTION
**Pull request by rgolangh**
This change is necessary for a successful official build

### CODE REVIEW

File: Dockerfile.rhel [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+
+LABEL   com.redhat.component="machine-api" \
+        name="cluster-api-provider-kubevirt" \
+        architecture="x86_64" \
+        summary="cluster-api-provider-kubevirt" \
+        maintainer="Nir Argaman <nargaman@redhat.com>"
```
Comment (bardielle) [inline Dockerfile.rhel:20]: can we add multiple maintainers?

**Comment by rgolangh** [comment]
I think so, but the best is to create a mailing list like ocprhv@redhat.com.

**Pull request merged**